code/game/objects/items/weapons/storage/bags.dm

```diff
@@ -568,3 +568,24 @@ var/global/list/plantbag_colour_choices = list("plantbag", "green red stripe", "
 	storage_slots = 3
 	w_class = W_CLASS_LARGE
 	slot_flags = SLOT_BELT | SLOT_POCKET
+
+
+
+// -----------------------------
+//          Xenobiology Bag
+// -----------------------------
+
+
+/obj/item/weapon/storage/bag/xenobio
+	icon = 'icons/obj/storage/storage.dmi'
+	icon_state = "slimebag"
+	name = "extract bag"
+	desc = "A bag designed to hold slime extract and other slime-related products."
+	item_state = "satchel"
+	origin_tech = Tc_BLUESPACE + "=2;" + Tc_MATERIALS + "=2;" + Tc_ENGINEERING + "=2"
```

Comment:
Why does an ordinary bag provide a higher level of bluespace tech than the subspace tunneler or tracking beacon? It even gives equal tech to an artificial bluespace crystal. Why does this give bluespace tech at all?

Reply:
This. Why does it have bluespace? I can always appreciate things having at least a tech value but I don't see the reason for that one.

Reply:
I was probably tired and didn't think much about the research levels. Bluespace tech is silly, yeah. I removed bluespace educed the tech requirement for the other categories since it is just a bag after all.